openrouteservice/src/main/java/heigit/ors/api/requests/routing/RouteRequestHandler.java

```diff
@@ -99,6 +99,25 @@ public  RoutingRequest convertRouteRequest(RouteRequest request) throws StatusCo
             }
         }
 
+        if (request.hasSkipSegments()) {
+            String skipSegmentsName = "skip_segments";
```

Comment:
As mentioned above - there should be a constant within the API's RouteRequest object that defines the name of the skip segments property